docs/source/getting_started/freesurfer.md

```diff
@@ -0,0 +1,69 @@
+To perform inverse modeling, or also called source estimation or source
+localization, we need to ensure that a couple prerequisities are met.
+Essentially, starting from a collection of 2-dimensional MRI images of coronal,
+axial, and saggital slices of a participant's head, we need to construct a
+3-dimensional representation of the brain, skull, and scalp. Furthermore,
+it's highly advantegous to attach labels to different brain areas according
+to common anatomical atlases, so that we could, for example, restrict
+subsequent analyses to specific cortical regions, and compare activation
+in these regions across participants.
+
+BIDS raw datasets, however, do **not** include any of these 3D representations
+and parcellations. (Note that, however, these *derivatives* **are** sometimes
+distributed along with a datasets inside a `derivatives/` folder). Instead,
+they ship e.g. with T1-weighted images only (and, sometimes, include FLASH
+images too).
+
+## Install FreeSurfer
+
+Before running the source-analysis parts of the Study Template, you need to
+create said 3D surfaces and parcellations. This is done using the tool
```

Comment:
Said?

Reply:
Past of "say" :) as in, "previously mentioned"
Will rephrase as it doesn't seem to be clear!